docs/sources/helm-charts/mimir-distributed/configure/configure-gem-federation-frontend.md

```diff
@@ -0,0 +1,155 @@
+     extraEnvFrom:
+       - secretRef:
+           name: gem-tokens
+
+   # Configure the remote GEM clusters to query.
+   mimir:
+     structuredConfig:
+       auth:
+         type: trust
+       # The federation-frontend doesn't handle authentication and authorization. Disabling multitenancy means the federation-frontend will not require the X-Scope-OrgID header.
+       # With disabled multitenant the federation-frontend will use the authn/z material from the proxy_targets configuration.
+       # For a complete list of configuration options, refer to the configuration reference at https://grafana.com/docs/enterprise-metrics/<GEM_VERSION>/config/reference/#federation.
+       multitenancy_enabled: false
+       federation:
+         proxy_targets:
+           - name: "cluster-local-gem"
+             url: "http://gem-query-frontend.monitoring.svc.cluster.local:8080/prometheus"
+             basic_auth:
+               username: tenant-1
+               password: "${CLUSTER_LOCAL_GEM_PASSWORD}"
+
+           - name: "cluster-remote-gem"
+             url: "https://gem.monitoring.acme.local/prometheus"
+             basic_auth:
+               username: tenant-2
+               password: "${CLUSTER_REMOTE_GEM_PASSWORD}"
+
+   # Disable all other components
+   distributor:
+     enabled: false
+   admin_api:
+     enabled: false
+   alertmanager:
+     enabled: false
+   ingester:
+     enabled: false
+   gateway:
+     enabled: false
+   querier:
+     enabled: false
+   query_frontend:
+     enabled: false
+   query_scheduler:
+     enabled: false
+   compactor:
+     enabled: false
+   store_gateway:
+     enabled: false
+   ruler:
+     enabled: false
+   overrides_exporter:
+     enabled: false
+
+   # Disable all caches
+   admin-cache:
+     enabled: false
+   chunks-cache:
+     enabled: false
+   index-cache:
+     enabled: false
+   metadata-cache:
+     enabled: false
+   results-cache:
+     enabled: false
+
+   # Disable MinIO
+   minio:
+     enabled: false
+
+   # The federation-frontend doesn't need the rollout-operator for rollouts, so it can be disabled.
+   rollout_operator:
+     enabled: false
+
+   tokengenJob:
+     enable: false
+   ```
+
+4. Deploy the Federation-Frontend using `helm`:
+
+   ```bash
+   helm install federation-frontend grafana/mimir-distributed -f federation-frontend.yaml -n federation-frontend-demo
+   ```
+
+   This will deploy only the Federation-Frontend component. The Federation-Frontend will be configured to proxy queries to the GEM clusters specified in the `proxy_targets` configuration.
+
+5. Verify that the Federation-Frontend is running. The simplest way to do this is to issue a label names query against the Federation-Frontend service.
+
+   This example tries to reach the Kubernetes service from within the cluster.
+   ```bash
+   curl -XPOST 'https://mimir-federation-frontend:8080/prometheus/api/v1/labels' -d 'start=2024-01-01T00:00:00.0Z' -d 'end=2025-01-01T00:00:00.0Z'
```

Comment:
This will query all label names for the year 2024 - is there some way we can construct this so it queries less data and also won't become outdated? (eg. imagine someone running this in 2026 when they've only just stood up the target clusters)

Reply:
if we fix the bug for label values, then we can just not specify `start` and `end`. Let's keep this open to see how complicated it will be to fix. If we can fix it in time, i'll just drop the start and end from here

Reply:
ok, i just added a command which generates always queries the last 1 year; see 87bebecf0e1dc2acaa1fd5d4a96d60e8a2ad7989